pkg/chains/formats/intotoite6/intotoite6_test.go

```diff
@@ -66,10 +66,10 @@ func TestCreatePayload1(t *testing.T) {
 			},
 			Invocation: slsa.ProvenanceInvocation{
 				Parameters: map[string]string{
```

Comment:
I wonder if we should use `map[string]ArrayOrString` (https://pkg.go.dev/github.com/tektoncd/pipeline@v0.36.0/pkg/apis/pipeline/v1beta1#ArrayOrString) here instead and not bother with the string encoding. 🤔  wdyt?

Reply:
Yeah, that's a better approach. I'll give that a shot.

Reply:
Done. Here's an example from a dummy task I used:
```
🐚 cosign download attestation quay.io/lucarval/minimal-container:latest | jq '.payload | @base64d | fromjson | .predicate.invocation.parameters'
{
  "BUILDER_IMAGE": "quay.io/buildah/stable:v1.18.0",
  "BUILD_EXTRA_ARGS": "",
  "CONTEXT": ".",
  "DOCKERFILE": "./Dockerfile",
  "FORMAT": "oci",
  "IMAGE": "quay.io/lucarval/minimal-container:latest",
  "PUSH_EXTRA_ARGS": "",
  "SKIP_PUSH": "false",
  "STORAGE_DRIVER": "vfs",
  "SUPER_ARRAY": [
    "hello",
    "array"
  ],
  "TLSVERIFY": "true"
}
```